CRM/Eventsextras/Form/Settings.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+use CRM_Eventsextras_ExtensionUtil as E;
```

Comment:
@tunbola   I think we should leave it as it is as it is CiviCRM standard.

Reply:
What I am saying is that we can alias it into something clear and readable rather than just `E`. What we need to do is to do something like `CRM_Eventsextras_ExtensionUtil as ExtensionUtil` and the  wherever `E` is used in the code replace with `ExtensionUtil`. The underlying class has not changed.

Reply:
@tunbola  understand your point but I believe we should be leaving it as it as it is CiviCRM generated and I have seen that we are maintaining the ExtensionUtil as E in other project.